field/composite.go

```diff
@@ -425,6 +427,18 @@ func (f *Composite) unpackSubfieldsByTag(data []byte) (int, error) {
 		}
 		tag := string(tagBytes)
 		if _, ok := f.spec.Subfields[tag]; !ok {
+			if f.skipUnknownTLVTags() {
+				// Obtain the length of the unknown tag and add it to the offset.
+				// Because BER-TLV lengths are decoded dynamically, the maxLen method argument is ignored.
+				const ignoredMaxLen int = 0
```

Comment:
let's move definition of `ignoredMaxLen` out of the method (it can be line 415) so we don't define const for each unpack.